packages/core/ui-settings/core-ui-settings-server-internal/src/routes/index.ts

```diff
@@ -12,9 +12,9 @@ import { registerGetRoute } from './get';
 import { registerSetManyRoute } from './set_many';
 import { registerSetRoute } from './set';
 
-export function registerRoutes(router: InternalUiSettingsRouter) {
-  registerGetRoute(router);
-  registerDeleteRoute(router);
-  registerSetRoute(router);
-  registerSetManyRoute(router);
+export function registerRoutes(router: InternalUiSettingsRouter, publicApiEnabled: boolean = true) {
```

Comment:
`publicApiEnabled` being optional is misleading. Jus make the option mandatory and adapt existing usages (if not done already)

Reply:
This argument is no longer needed after separating the public from the internal routes in different files.